EnglishFilter/sections/specific.txt

```diff
@@ -261,6 +261,7 @@ feetporno.com,familyporn.tv,freehardcore.com,webvoyeur.com,moviesand.com,momvids
 !
 !
 ||msn.com/images/*/960x300_$domain=lovehug.net
+katmoviehd4.com##[placeholder]
```

Comment:
What is this rule for? You block some useful elements with it.

Reply:
@zloyden Ah, yes. It blocks a lot of useful thing but please check the link and the screenshot. `https://katmoviehd4.com/links/step-1.php`
<details>

![image](https://user-images.githubusercontent.com/76908410/111968713-6db3eb00-8b34-11eb-9002-30c4650f0f75.png)
</details>

Reply:
Try this `katmoviehd4.com##center > div#new > :not([role="button"])`, it should be ok.